charts/scalardb-graphql/templates/scalardb-graphql/service.yaml

```diff
@@ -16,3 +16,19 @@ spec:
       name: http
   selector:
     {{- include "scalardb-graphql.selectorLabels" . | nindent 4 }}
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "scalardb-graphql.fullname" . }}-metrics
+  labels:
+    {{- include "scalardb-graphql.labels" . | nindent 4 }}
+spec:
+  ports:
+  - name: scalardb-graphql-prometheus
+    port: 9080
+    targetPort: 9080
```

Comment:
The Prometheus port `9080` differs from `8080` on other charts because `8080` is used as the default port of the GraphQL HTTP server. I feel it was a design mistake. I wonder if we should change the GraphQL HTTP port to another number such as `8000`.

Reply:
This is a minor comment for your reference. This comment does not prevent for merging the PR.

I think the GraphQL HTTP port is used for GraphiQL that is accessed from browsers as follows.

* GraphiQL URL
  ```
  http://localhost:8080/graphql
  ```

Also, the default port of Web UI of Scalar Manager is `8000` now.
The Scalar Manager Web UI is accessed from browsers as follows.
* Scalar Manager Web UI URL
  ```
  http://localhost:8000/
  ```

So, if we change the default port of GraphQL HTTP to `8000`, I think it causes another conflict of port between the GraphiQL and the Scalar Manager Web UI.
Basically, I think both will be exposed to the outside of Kubernetes Cluster to access them from browsers.

I think we need to consider the above thing when we decide the default port.